src/kit.ts

```diff
@@ -809,7 +809,7 @@ async function scanDirForClangForMSVCKits(dir: string, vsInstalls: VSInstallatio
                     clangKits.push({
                         name: clangKitName,
                         visualStudio: kitVSName(vs),
-                        visualStudioArchitecture: vs_arch,
+                        visualStudioArchitecture: generatorPlatformFromVSArch[vs_arch] as string || vs_arch,
```

Comment:
This line is incorrect.  You only do it for the `platform`.  This should stay as `vs_arch`